components/accelerator/nvidia/query/infiniband/ibstat.go

```diff
@@ -281,5 +283,8 @@ func ParseIBStat(input string) (IBStatCards, error) {
 	if err := yaml.Unmarshal([]byte(txt), &cards); err != nil {
 		return nil, err
 	}
+	if len(cards) == 0 {
+		return nil, ErrIbstatOutputEmpty
```

Comment:
hmm... are we sure the output result is empty? or just we do not parse any cards? these two can be different. better to use different error message